### PR TITLE
Updating WebAuthn and biometrics to passkeys

### PR DESCRIPTION
**Pull request by bradmccarty**
These updates are all on the blog. I'll focus on articles on the next pass.

### CODE REVIEW

File: astro/src/content/blog/authenticators-ceremonies-webauthn-oh-my.mdx [hunk 1]
```diff
@@ -9,38 +9,38 @@ tags: webauthn, passkeys, passwordless
...
 
-## What Is WebAuthn
+## What Are Passkeys?
 
 User’s passwords are often weak, reused, and can be easily hijacked. The site ["have i been pwned?"](https://haveibeenpwned.com) has over 11 billion accounts that have been compromised. Because of this, they're not suited for guarding sensitive accounts like banking or social media applications. While MFA offers additional security, WebAuthn is even better when it comes to guarding sensitive data since it uses passwordless authentication.
 
-WebAuthn provides a means for servers to authenticate users using cryptographic keys (private and public) in place of passwords. The private and public keys are used to authenticate users, and the public key is stored in the server (also known as a relying party or RP), while the private key is stored locally in the user's device.
+Passkeys provide a means for servers to authenticate users using cryptographic keys (private and public) in place of passwords. The private and public keys are used to authenticate users, and the public key is stored in the server (also known as a relying party or RP), while the private key is stored locally in the user's device.
```
Comment (emilyjans) [inline astro/src/content/blog/authenticators-ceremonies-webauthn-oh-my.mdx:24]: This swap doesn't work. 

Passkeys are credentials that allow a user to prove their identity, while WebAuthn is a standard javascript API that allows an application to create and use passkeys for sign-in. This is specifically talking about WebAuthn and not passkeys.

File: astro/src/content/blog/authenticators-ceremonies-webauthn-oh-my.mdx [hunk 2]
```diff
@@ -9,38 +9,38 @@ tags: webauthn, passkeys, passwordless
...
+* Passkeys are easy to implement because it consists of [browser APIs that can be easily used by developers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API#examples).
 * It offers a smooth authentication experience for users.
 
-## How WebAuthn Works
+## How Passkeys Work
 
-WebAuthn consists of three components: the authenticator (the user's device), the web browser, and the server (RP). In essence, the user initiates a login via the browser, the browser then prompts the user to authenticate using their device (via fingerprint or facial recognition, for example), and then if successful, the user is logged into the web application.
+Passkey authentication consists of three components: the authenticator (the user's device), the web browser, and the server (RP). In essence, the user initiates a login via the browser, the browser then prompts the user to authenticate using their device (via fingerprint or facial recognition, for example), and then if successful, the user is logged into the web application.
```
Comment (emilyjans) [inline astro/src/content/blog/authenticators-ceremonies-webauthn-oh-my.mdx:43]: Same with this one. The swap doesn't work here. 


Passkey = credential but this describes WebAuthn as a whole

**Review by emilyjans — CHANGES_REQUESTED** [review]
Let a few individual comments. 

Most of the swaps are ok, but a few don't make sense.

File: astro/src/content/blog/authenticators-ceremonies-webauthn-oh-my.mdx (hunk 1 above)
Comment (bradmccarty) [reply astro/src/content/blog/authenticators-ceremonies-webauthn-oh-my.mdx:24]: Reverted

File: astro/src/content/blog/authenticators-ceremonies-webauthn-oh-my.mdx (hunk 2 above)
Comment (bradmccarty) [reply astro/src/content/blog/authenticators-ceremonies-webauthn-oh-my.mdx:43]: Reverted

**Review by emilyjans — APPROVED** [review]
ltgm

**Pull request merged**